docs/processes/uni-up.md

```diff
@@ -0,0 +1,79 @@
+<!--
+© 2021 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# Unicode Update
+
+*[This page's last thorough update was on [5 August
+2010](https://sites.google.com/site/icusite/system/app/pages/admin/revisions?wuid=wuid:gx:6c0edddbaea77d12).]*
+
+The International Components for Unicode (ICU) implement the Unicode Standard
+and many of its Standard Annexes faithfully and are updated to a new Unicode
+version soon after its release. Usually, the ICU team participates in the
+Unicode beta process by updating to a new Unicode version in a branch and
+testing it thoroughly. In the past, this has uncovered problems that could be
+fixed before the release of the new Unicode version.
+
+One notable exception to ICU implementing all of Unicode is that it does not
+provide any access to Unihan data (mostly because of low demand and the large
+size of the Unihan data).
+
+## Update process
+
+For the last several updates, there is a [change log
+here (BROKEN LINK)](http://source.icu-project.org/repos/icu/icu/trunk/source/data/unidata/changes.txt).
+In short, most of the ucd .txt files are copied into the ICU repository, either
+without modification or, for some files, with comments removed and lines merged
+to reduce their size.
+
+Some of the data files are not part of the Unicode release but are output from
+Mark's Unicode tools. See <http://sites.google.com/site/unicodetools/>
+
+Note: We have looked at using the [UCD
+XML](http://www.unicode.org/ucd/#UCDinXML) files, but did not want to rely on
+them alone until there was a way to verify that they contain precisely the same
+data as the .txt files. Also, using the XML files would require a partial
+rewrite of the existing tools. (There was an outdated, experimental, partial UCD
+XML parser at this now-broken (as of 2021) link:
+<http://bugs.icu-project.org/trac/browser/icuhtml/trunk/design/properties/genudata>)
+
+The ICU Unicode tools parse the text files, process the data somewhat, and write
+binary data for runtime use. Most of these tools live in a [source
+tree (BROKEN LINK)](http://bugs.icu-project.org/trac/browser/tools/trunk/unicode) separate
```

Comment:
https://github.com/unicode-org/icu/tree/master/tools/unicode

still somewhat separate from ICU4C/ICU4J, but no longer in a separate repo

Reply:
Done.